rollup.config.js

```diff
@@ -0,0 +1,39 @@
+import babel from "@rollup/plugin-babel";
+import resolve from "@rollup/plugin-node-resolve";
+import external from "rollup-plugin-peer-deps-external";
+import commonjs from "@rollup/plugin-commonjs";
+import terser from "@rollup/plugin-terser";
+import typescript from "@rollup/plugin-typescript";
+
+const extensions = [".js", ".jsx", ".ts", ".tsx"];
+export default [
+  {
```

Comment:
@JesseKuntz does this config export out types?  If not, I can add a PR that does this based off of work we did at the offsite.

Reply:
Not yet! I was planning on following up with that, but that's right, since you already had that working feel free to open a PR once this is merged!